### ISSUE TITLE
Basic HTTP auth reloading page logs out

### ISSUE THREAD

**Issue by 1am** [partkeepr/PartKeepr#634]
Hi,

I'm currently using Basic HTTP authentication to access PartKeepr and I've noticed that after being successfully logged in and using the system for a while and then reloading web page it forces me to log in again. Is that something which is done on purpose or should I look at my server configuration more in depth?

Best regards,
PB

**Comment by felicitus** [comment]
Yes, that's by design. Reloading the page currently requires re-login. I'll keep this issue open as a feature request.

**Comment by nitrocaster** [comment]
+1. It's really annoying to lose the context after reloading the page.
